src/NUnitFramework/framework/Internal/Results/TestResult.cs

```diff
@@ -30,17 +30,15 @@
 using System.Reflection;
 using System.Text;
 using System.Threading;
-using NUnit.Framework.Interfaces;
-#if NET20
 using NUnit.Compatibility;
-#endif
+using NUnit.Framework.Interfaces;
 
 namespace NUnit.Framework.Internal
 {
     /// <summary>
     /// The TestResult class represents the result of a test.
     /// </summary>
-    public abstract class TestResult : ITestResult
+    public abstract class TestResult : LongLivedMarshalByRefObject, ITestResult
```

Comment:
What prompted this change?

Reply:
I had the same question. 🙂

Reply:
One of the tests in `LowTrustFIxture` was throwing an exception when trying to access the test result across a remoting boundary. If we ever implement a facility to isolate a test or fixture in a separate AppDomain, we'll probably find more things like this to fix.